Tests/Expected/Strings/structured-swift2-context-defaults-no-comments.swift

```diff
@@ -0,0 +1,86 @@
+// Generated using SwiftGen, by O.Halligon — https://github.com/SwiftGen/SwiftGen
+
+import Foundation
+
+// swiftlint:disable file_length
+
+// swiftlint:disable explicit_type_interface identifier_name line_length nesting type_body_length type_name
+enum L10n {
+  static let AlertMessage = L10n.tr("alert_message")
+  static let AlertTitle = L10n.tr("alert_title")
+  static func ObjectOwnership(p1: Int, p2: String, p3: String) -> String {
+    return L10n.tr("ObjectOwnership", p1, p2, p3)
+  }
+  static func Private(p1: String, p2: Int) -> String {
+    return L10n.tr("private", p1, p2)
+  }
+
+  enum Apples {
+    static func Count(p1: Int) -> String {
+      return L10n.tr("apples.count", p1)
+    }
+  }
+
+  enum Bananas {
+    static func Owner(p1: Int, p2: String) -> String {
+      return L10n.tr("bananas.owner", p1, p2)
+    }
+  }
+
+  enum Settings {
+
+    enum NavigationBar {
+      static let `Self` = L10n.tr("settings.navigation-bar.self")
+
+      enum Title {
+
+        enum Deeper {
+
+          enum Than {
+
+            enum We {
+
+              enum Can {
+
+                enum Handle {
+
+                  enum No {
+
+                    enum Really {
+
+                      enum This {
+
+                        enum Is {
+                          static let Deep = L10n.tr("settings.navigation-bar.title.deeper.than.we.can.handle.no.really.this.is.deep")
```

Comment:
<img width="400" alt="We need to go deeper" src="http://i1.kym-cdn.com/photos/images/newsfeed/000/531/557/a88.jpg">